training-watchdog/src/constants.py

```diff
@@ -0,0 +1,14 @@
+USER_TRAINING_IMAGE_COUNT = 30
+FILE_SYSTEM_POLLING_DELAY_IN_SECONDS = 2
+already_trained_students_count = -1
```

Comment:
Could we capitalize this variable for uniformity?

Reply:
the value of this variable changes at runtime, it's not a constant... 
shall I still capitalize it?

Reply:
Let's capitalize and add a comment here to specify the value will change at run-time.

Reply:
ohkay... :)